### DISCUSSION TITLE
How to generate the GraphML after the indexing process?

### DISCUSSION THREAD

**Discussion by tinghao-ai** [microsoft/graphrag#408] [Q&A]
tired "poetry run python -m graphrag.index --config settings.yaml --resume 20240707-023656 -v" but didn't work, it's not in the workflow process right?

**Comment by natoverse** [comment]
If you turn on snapshot.graphml flag in the [settings.yml](https://microsoft.github.io/graphrag/posts/config/env_vars/) and just re-run the index process (no need to use resume), it will pull all of the results from the cache without recomputing, but the graphml snapshots will also be created.
